metadata

```diff
@@ -1 +1 @@
-Subproject commit 86952e2535a92dccdb6c61ef15baf1f40add7966
+Subproject commit 7e3d55a19b1535b1fc4da03807d84d6399888d74
```

Comment:
Hm. Did you update the sub-projects? I'd rather have that happen in a separate commit. It might be enough to rebase onto master to get rid of these extra changes.